client/src/app/site/motions/components/motion-detail/motion-detail.component.html

```diff
@@ -238,7 +238,7 @@ <h3 translate>Category</h3>
         <!-- Origin -->
         <div *ngIf="motion && motion.origin || editMotion">
             <div *ngIf='!editMotion'>
-                <h3 translate> Origin</h3>
+                <h4 translate> Origin</h4>
```

Comment:
whitespace should be removed I suppose
```suggestion
                <h4 translate>Origin</h4>
```

Reply:
done